sync-diff-inspector/shard-diff.md

```diff
@@ -6,11 +6,15 @@ aliases: ['/docs/dev/sync-diff-inspector/shard-diff/','/docs/dev/reference/tools
 
 # Data Check in the Sharding Scenario
 
-sync-diff-inspector supports data check in the sharding scenario. Assuming that you have two MySQL instances and now use a replication tool to replicate data into TiDB as shown below:
+sync-diff-inspector supports data check in the sharding scenario. Assuming that you have multiple MySQL instances using a replication tool to replicate data into TiDB, and now you can use sync-diff-inspector to check upstream and downstream data.
```

Comment:
```suggestion
sync-diff-inspector supports data check in the sharding scenario. Assuming that you use the DM replication tool to replicate data from multiple MySQL instances into TiDB, and now you can use sync-diff-inspector to check upstream and downstream data.
```
It's "**you** use", not "the **multiple MySQL instances** use".

Reply:
got it. thx:)